apollo-router/src/axum_factory/axum_http_server_factory.rs

```diff
@@ -329,7 +329,14 @@ async fn handle_graphql(
     service: router::BoxService,
     http_request: Request<Body>,
 ) -> impl IntoResponse {
-    match service.oneshot(http_request.into()).await {
+    let request: router::Request = http_request.into();
+    let context = request.context.clone();
+
+    let res = service.oneshot(request).await;
+    let overhead_seconds = (context.busy_timer.lock().await.current() / 1000_000_000) as f64;
+    tracing::info!(histogram.apollo_router_overhead_time = overhead_seconds,);
```

Comment:
To reflect the caveat better, should we call this `apollo_router_internal_execution_time` or `apollo_router_processing_time` or something along those lines?

Reply:
processing time sounds good, updated in [a7c30fd](https://github.com/apollographql/router/pull/2371/commits/a7c30fdfe4e35992068038752d2ed818d403dce6)